src/components/signup/SignupForm/FormRow/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Children, ReactElement } from 'react';
+
+import Text from '@/components/common/Text';
+
+import * as S from './style';
+import { FormRowProps } from './type';
+
+const FormRow = ({
+  id,
+  label,
+  isRequired = false,
+  children,
+  bottomText,
+  errorMessage,
+  direction = 'column',
+  ...props
+}: FormRowProps) => {
+  const isError = Children.toArray(children).some(
+    child => (child as ReactElement).props.error,
+  );
+
+  return (
+    <S.FormRow {...props}>
+      <S.Label htmlFor={id}>
+        <Text isRequired={isRequired}>{label}</Text>
+      </S.Label>
+      <S.Container>
+        <S.Content direction={direction}>{children}</S.Content>
+        <S.BottomTextWrapper>
+          {bottomText != null ? (
+            <S.BottomText>{bottomText}</S.BottomText>
+          ) : null}
+          {isError ? <S.ErrorMessage>{errorMessage}</S.ErrorMessage> : null}
```

Comment:
```suggestion
          {errorMessage && <S.ErrorMessage>{errorMessage}</S.ErrorMessage>}
```
이렇게 errorMessage가 있는 경우를 에러가 있다고 생각하면
isError가 필요없습니다.

Reply:
아래 리뷰와 함께 묶어서 질문드리겠습니다!!